content/help/addons/hero-image.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Hero Image
+tags:
+- Addons
+- Hero Image
+- Image upload
+category: help
+menu:
+  help:
+    parent: addons
+    identifier: addon-hero-image
+versioning:
+  added: 2.6
+---
+
+## Overview
+
+{{% cloudfeature %}}
+
+This plugin allows you to:
+
+- Associate an background image with a category.
+- Inherit the parent categories image if one isn't set.
+- Set a "top-level" image that everything falls back to if no image is found on a parent category.
+- Access this image through a [Smarty Template]() or [Customize Theme]().
+
+## Setup
+
+- Visit the addon's page of the dashboard.
+- Enable the HeroImage plugin.
+
+![](/img/help/addons/hero-image/hero-image-settings.png)
+
+## Set the default image
+
+1. Click on the settings icon next to the plugin.
+2. Click on the file picker to select your image.
+![](/img/help/addons/hero-image/hero-image-set-default.png)
+3. Click save.
+
+## Set a category image.
```

Comment:
Maybe reframe the category image as an optional overwrite instead of a fallback.